### ISSUE TITLE
Article Add Error in Blog Authentication

### ISSUE THREAD

**Issue by eyoung100** [cakephp/docs#3792]
If you follow Part 3, and add categories, the `ArticlesController.php` file contains:

```
 // Just added the categories list to be able to choose
// one category for an article
$categories = $this->Articles->Categories->find('treeList');
$this->set(compact('categories'));
```
The rewritten add function in the Authentication tutorial is missing the above snippet, which creates an empty dropdown/view mismatch.

**Comment by lorenzo** [comment]
Would you like to submit the changes yourself? There is a "Improve this page" button in the docs that will take you to an editor

**Comment by eyoung100** [comment]
PR submitted for review....

**Comment by markstory** [comment]
Fixed in #3794 Thank you!
